packages/framework-info/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "noEmit": true /* Specify an output folder for all emitted files. */
```

Comment:
Vite handles transpilation of TS files by default so this is left to allow for type checking prior to building with `vite`